rasa_sdk/executor.py

```diff
@@ -116,6 +116,14 @@ def utter_custom_json(self, json_message, **kwargs):
 
         self.messages.append(json_message)
 
+    def utter_image_url(self, image, **kwargs):
+        """ sends image url to the output channel as image"""
+
+        json_message = {"image": image}
```

Comment:
please change these `json_message` to `message` (`utter_custom_json` is a little special -- the rest are just `message`s.)